core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -416,6 +417,11 @@ void deactivateTariffTest() {
             .andExpect(status().isOk());
     }
 
+    @Test
+    void deactivateCourier() throws Exception {
+        mockMvc.perform(patch(ubsLink + "/deactivateCourier/{id}", 1L)).andExpect(status().isOk());
```

Comment:
please verify that `superAdminService.deactivateCourier(id)` was called with the expected `id`